### PR TITLE
Add properties to SvgMask and SvgElement

### LINKED ISSUE #610: Add font-stretch property to SvgElement
#### Description

Value: | normal \| wider \| narrower \|ultra-condensed \| extra-condensed \|condensed \| semi-condensed \|semi-expanded \| expanded \|extra-expanded \| ultra-expanded \| inherit
-- | --
Initial: | normal
Applies to: | text content elements
Inherited: | yes
Percentages: | N/A
Media: | visual
Animatable: | yes

https://www.w3.org/TR/SVG11/text.html#TextElement

### PR DESCRIPTION
**Pull request by wieslawsoltes**
#### Reference Issue

Fixes #610

#### What does this implement/fix? Explain your changes.

* added MaskUnits, MaskContentUnits, X, Y, Width and Height properties to SvgMask
https://www.w3.org/TR/SVG11/masking.html#MaskElement

* added FontStretch property to SvgElement
https://www.w3.org/TR/SVG11/text.html#FontStretchProperty

### CODE REVIEW

**Comment by H1Gdev** [comment]
@wieslawsoltes 

Is this just adding attributes ?
Do you plan to implement this feature in future PRs ?

**Comment by wieslawsoltes** [comment]
This PR only adds attributes.

czw., 30 sty 2020, 03:25 użytkownik H1Gdev <notifications@github.com>
napisał:

File: Source/SvgElementStyle.cs [hunk 1]
```diff
@@ -238,6 +238,16 @@ public virtual SvgFontWeight FontWeight
             set { Attributes["font-weight"] = value; IsPathDirty = true; }
         }
 
+        /// <summary>
+        /// Indicates the desired amount of condensing or expansion in the glyphs used to render the text.
+        /// </summary>
+        [SvgAttribute("font-stretch")]
+        public virtual SvgFontStretch FontStretch
+        {
+            get { return GetAttribute("font-stretch", true, SvgFontStretch.Inherit); }
```
Comment (H1Gdev) [inline Source/SvgElementStyle.cs]: `font-stretch` initial value is `normal`.
So, 3rd argument is `SvgFontStretch.Normal`.

Comment (wieslawsoltes) [reply Source/SvgElementStyle.cs]: Isn't the `SvgFontStretchConverter` handling default case?
```
public SvgFontStretchConverter() : base(SvgFontStretch.Normal) { }
```

Comment (H1Gdev) [reply Source/SvgElementStyle.cs]: default value of property, not  converter.

Comment (H1Gdev) [reply Source/SvgElementStyle.cs]: Some users seem to build SVG programmatically.

Comment (wieslawsoltes) [reply Source/SvgElementStyle.cs]: Agree.

**Pull request merged**